.github/workflows/osrm-backend.yml

```diff
@@ -558,7 +554,7 @@ jobs:
         fi
         popd
     - name: Build example
-      if: ${{ matrix.NODE_PACKAGE_TESTS_ONLY != 'ON' }}
+      if: ${{ matrix.NODE_PACKAGE_TESTS_ONLY != 'ON' && matrix.ENABLE_CONAN != 'ON' }}
```

Comment:
No a lot of reasons to support building libosrm example with Conan... IMO if we want to use Conan deps there instead of system ones we should make libosrm Conan package itself...